code/create_app.py

```diff
@@ -436,6 +436,7 @@ def speech_config():
             if response.status_code == 200:
                 return {
                     "token": response.text,
+                    "key": speech_key,
```

Comment:
Is this required? We already have an endpoint in `get_speech_key` which returns the short lived key.

Reply:
reverted the made changes